addons/l10n_it_edi/views/report_invoice.xml

```diff
@@ -2,12 +2,21 @@
 <odoo>
     <template id="report_invoice_document" inherit_id="account.report_invoice_document">
         <div name="comment" position="before">
+            <t t-if="o.l10n_it_stamp_duty"><b>Stamp Duty: </b><span t-field="o.l10n_it_stamp_duty"></span><br/></t>
+            <t t-if="o.l10n_it_ddt_id"><b>Transport Document: </b><span t-field="o.l10n_it_ddt_id"></span><br/></t>
             <div t-if="o.l10n_it_origin_document_type" name="pa_fields">
                 <b><span t-field="o.l10n_it_origin_document_type"/></b>: <span t-field="o.l10n_it_origin_document_name"/><br/>
                 <t t-if="o.l10n_it_origin_document_date"><b>Document Date: </b><span t-field="o.l10n_it_origin_document_date"/><br/></t>
                 <t t-if="o.l10n_it_cig"><b>CIG: </b><span t-field="o.l10n_it_cig"/><br/></t>
                 <t t-if="o.l10n_it_cup"><b>CUP: </b><span t-field="o.l10n_it_cup"/><br/></t>
             </div>
         </div>
+
+        <p name="payment_communication" position="inside">
+            <t t-if="o.company_id.country_id.code == 'IT'">
```

Comment:
```suggestion
            <t t-if="o.country_code == 'IT'">
```
It prevents to wrongly use the country instead of the fiscal one. For reference:
```py
country_code = fields.Char(related='company_id.account_fiscal_country_id.code', readonly=True)
```